src/views/auth/Signout.vue

```diff
@@ -15,9 +12,9 @@ import SbcSignout from 'sbc-common-components/src/components/SbcSignout.vue'
   }
 })
 export default class Signout extends Vue {
-  /** The BCROS Home URL string. */
-  get bcrosHomeUrl (): string {
-    return sessionStorage.getItem('BUSINESSES_URL')
+  get redirectUrl (): string {
+    // after signout, redirect to BC Registries login page
+    return sessionStorage.getItem('REGISTRY_HOME_URL') + 'login'
```

Comment:
Better to give fall back also, if no value in session

Reply:
Generally, yes... In this case, it's an environment variable set when the app starts up. That would be a serious error that would manifest in many other things not working before the user gets to this signout page.

Also, a fallback would also need to use an environment variable, which also might not work.

So, I'm thinking, this is not something we need to worry about. What do you think?